.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -372,6 +372,17 @@ mkdir -p ${SUBMITTY_INSTALL_DIR}/library
 chown submitty_php:submitty_php ${SUBMITTY_INSTALL_DIR}/library
 chmod g+rwx ${SUBMITTY_INSTALL_DIR}/library
 
+# Create test folders
```

Comment:
No don't do this! Create any test folders in your tests. Nobody needs these laying around all of the time. Or wants them.